.ci/pipelines/thumbnails4j.groovy

```diff
@@ -19,23 +19,25 @@
  *
  */
 
+
 // Loading the shared lib
-@Library(['estc', 'entsearch']) _
+@Library('estc') _
 
-// Calling the pipeline against the `rubocop` stage
-eshPipeline(
-    timeout: 45,
-    project_name: 'Thumbnails4j',
-    repository: 'thumbnails4j',
-    stage_name: 'Thumbnails4j Unit Tests',
-    stages: [
-      [
-          name: 'Maven Build',
-          type: 'sh',
-          label: 'Maven Build',
-          script: './mvnw clean verify',
-          match_on_all_branches: true,
-      ]
-    ],
-    slack_channel: 'workplace-search-connectors'
-)
\ No newline at end of file
+pipeline {
+    agent { label('linux && immutable') }
```

Comment:
If you use the plain pipeline sequence, you might need to provide the `options` and `parameters` parts (see ent-search packaging's pipeline as an example)